tests/test_react_auth_defs.py

```diff
@@ -0,0 +1,119 @@
+import os
+from random import randrange
+import time
+import uuid
+from foursight_core.react.api.auth import Auth
+from foursight_core.react.api.envs import Envs
+from foursight_core.react.api.jwt_utils import jwt_encode
+
+AUTH0_CLIENT_ID = str(uuid.uuid4())
+AUTH0_SECRET = str(uuid.uuid4())
+AUTH0_SECRET_INVALID = str(uuid.uuid4())
+KNOWN_ENV_A = {
+    "name": "env-a",
+    "full_name": "env-a-full-name",
+    "short_name": "env-a-short-name",
+    "public_name": "env-a-public-name",
+    "foursight_name": "env-a-foursight-name"
+}
```

Comment:
Vast majority of what you want in this file should probably be `pytest.fixture`. https://docs.pytest.org/en/6.2.x/fixture.html

Reply:
Oh, hm, will take a look, thanks.